### ISSUE TITLE
Auth connection error

### ISSUE THREAD

**Issue by michrf** [einsteinx2/iSub#58]
Hi,
Can't connect to  server running  subsonic 4.9+ MusicCabinet plugin, version 0.7.24c.

The error is: 
Error: Optional (Error Domain=SubsonicErrorDomain Code=10 "Required parameter is missing." 
Userinfo={NSLocalizedDescription=Required parameter is missing.})
![error_isub](https://cloud.githubusercontent.com/assets/7806020/22401111/2ced4938-e5c8-11e6-9291-8ced97a13107.png)

**Comment by einsteinx2** [comment]
Looks like a MusicCabinet issue. I'll check with your server.

**Comment by einsteinx2** [comment]
So I took a look into MusicCabinet and unfortunately it looks like it was forked off of Subsonic 4.7, which is all the way back at Subsonic API version 1.8.0. iSub 4 supports only API version 1.13.0 and higher. The reason authentication is failing is the way of sending passwords changed to be more secure (using a salted hash).

It looks like MusicCabinet is a dead project with the last commits 4 years ago, so I doubt they’ll be updating any time soon.

Even if I fixed the sign in, it wouldn’t have various API calls that iSub 4 uses as they are newer than that.

So Subsonic 5.3 and above and Madsonic are supported, but unfortunately I likely won’t be able to support MusicCabinet unless someone updates it to provide the newer APIs
